src/relay/backend/utils.h

```diff
@@ -410,6 +416,15 @@ inline bool IsCompileEngineCacheDisabled() {
       .value();
 }
 
+/*!
+ * \brief Get a shared optimization pass prefix between vm and graph executor.
+ *
+ * \param targets The device type to `Target` mapping.
+ * \param is_vm A boolean indicating if the passes are used for vm or graph executor.
+ * \return An array of passes.
+ */
+Array<Pass> GetPrefixOpts(const Map<tvm::Integer, tvm::Target>& targets, bool is_vm);
```

Comment:
Better to explain more clearly about what these passes will do, or come up with a better function name.
But I don't have suggestions from my point of view ...

Reply:
Thanks for review! I renamed it to `GetPassPrefix` and added more comments. Hopefully it gets clearer. :)